dbc2val/Dockerfile

```diff
@@ -30,7 +30,7 @@ ENV PATH="/opt/venv/bin:$PATH"
 RUN /opt/venv/bin/python3 -m pip install --upgrade pip \
     && pip3 install --no-cache-dir -r requirements.txt
 
-RUN pip3 install wheel && pip3 install scons && pip3 install pyinstaller && pip3 install patchelf && pip3 install staticx
+RUN pip3 install wheel scons && pip3 install pyinstaller patchelf==0.17.0.0 staticx
```

Comment:
```suggestion
RUN pip3 install wheel scons pyinstaller patchelf==0.17.0.0 staticx
```

This was left here

Reply:
Yeah it is intended. Otherwise the arm build fails because pyinstaller and staticx can not find scons and wheel packages